runtime/src/iree/schemas/spirv_executable_def.fbs

```diff
@@ -17,6 +17,9 @@ table SpirVExecutableDef {
   // A map of entry point ordinals to string names as used in the shader module.
   entry_points:[string];
 
+  // Required subgroup size for each entry point. 0 means no requirement.
+  subgroup_sizes:[uint32];
```

Comment:
Just double checking forward/backward compatibility - I don't think we need to update any version information to make changes to the schema files.

Reply:
It's been a while since I've edited these, but previous PRs did not bump versions. Maybe we get a runtime error if the schema changed? A readme in https://github.com/iree-org/iree/tree/main/runtime/src/iree/schemas would be helpful.

Reply:
Okay SG! Let me follow that then. :)